pkgs/development/python-modules/fuzzywuzzy/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildPythonPackage, python, fetchFromGitHub, python-Levenshtein, pycodestyle, hypothesis, pytest }:
+
+buildPythonPackage rec {
+  pname = "fuzzywuzzy";
+  version = "0.16.0";
+
+  src = fetchFromGitHub {
```

Comment:
Looked at https://pypi.org/project/fuzzywuzzy/0.17.0/ and I've found the test in the tarball :smile: 
So please use `fetchPypi`.